index.ts

```diff
@@ -97,7 +97,7 @@ export default class Analytics {
 
     this.flushAt = Math.max(flushAt, 1);
     this.flushInterval = flushInterval;
-    this.maxInternalQueueSize = maxInternalQueueSize;
+    this.maxQueueSizeInBytes = maxQueueSizeInBytes;
```

Comment:
i have not confirmed, but the docs mention that the max payload for a `batch` request is 4mb. We could `Math.Min(maxQueueSizeInBytes, 1024 * 2000)`. any pref on this?

Reply:
after testing, `4030341` bytes, or `3935` kb, seem to be the upper bounds. requests with a size of `4063108` bytes, or `3967`kb were throwing errors. I'm assuming the size differential between 3935kb and 4mb is due to the additional size of the headers/request that are included in the HTTP request. For now, I'll default the values to `3935`